tests/integration/kubernetes/confidential_common.sh

```diff
@@ -210,9 +210,10 @@ function create_coco_pod_yaml_with_annotations() {
 confidential_teardown_common() {
 	local node="$1"
 	local node_start_time="$2"
+	local policy_settings_dir="$3"
```

Comment:
Is it worth passing through the `policy_settings_dir` here as no callers of this function use it, so it's always an empty string, so we could just pass through empty string inside of this?